pkgs/development/ocaml-modules/opium/default.nix

```diff
@@ -0,0 +1,30 @@
+{ buildDunePackage
+
+, ppx_sexp_conv
+, ppx_fields_conv
+
+, cmdliner
+, cohttp-lwt-unix
+, logs
+, magic-mime
+, opium_kernel
+, stringext
+
+, alcotest
+}:
+
+buildDunePackage rec {
+	pname = "opium";
+	inherit (opium_kernel) version src meta minimumOCamlVersion;
```

Comment:
what is the difference between these two packages, one just seems to have more build inputs

Reply:
It's just how upstream has organized things. Following examples like [cohttp](https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/ocaml-modules/cohttp/lwt-unix.nix) where a single source is used in multiple derivations.